pentaho-pdi-platform/src/main/java/org/pentaho/platform/pdi/WebContextServlet.java

```diff
@@ -46,47 +44,55 @@
 public class WebContextServlet extends HttpServlet {
   // this is the map that groups contexts to a set of urls to add to the context
   private Map<String, Set<IPlatformWebResource>> contextResourcesMap = new HashMap<>();
-  protected static ICacheManager cache = PentahoSystem.getCacheManager( null );
 
   static final String WEB_CONTEXT_JS = "webcontext.js"; //$NON-NLS-1$
 
   static final String CONTEXT_PATH = "/";
   private static final String REQUIREJS_INIT_LOCATION = "requirejs-manager/js/require-init.js";
 
+  private static final String DEFAULT_SERVICES_ROOT = "cxf/";
```

Comment:
We were discussing this after you left and we had doubts about this, so we decided to merge and if tomorrow we find that the urls should go with '/' at the beginning, I will do a second round of commits to change it